ui/helpers/utils/multichain/blockExplorer.ts

```diff
@@ -0,0 +1,38 @@
+import { getAccountLink } from '@metamask/etherscan-link';
+import { KnownCaipNamespace, parseCaipChainId } from '@metamask/utils';
+import { MultichainNetwork } from '../../../selectors/multichain';
+import { MULTICHAIN_NETWORK_BLOCK_EXPLORER_URL_MAP } from '../../../../shared/constants/multichain/networks';
+import { normalizeSafeAddress } from '../../../../app/scripts/lib/multichain/address';
+
+export const getMultichainBlockExplorerUrl = (
+  network: MultichainNetwork,
+): string => {
+  const { namespace } = parseCaipChainId(network.chainId);
+  if (namespace === KnownCaipNamespace.Eip155) {
+    return network.network?.rpcPrefs?.blockExplorerUrl ?? '';
+  }
+
+  const multichainExplorerUrl =
+    MULTICHAIN_NETWORK_BLOCK_EXPLORER_URL_MAP[
+      network.chainId as keyof typeof MULTICHAIN_NETWORK_BLOCK_EXPLORER_URL_MAP
+    ] ?? '';
+
+  return multichainExplorerUrl;
```

Comment:
We should be able to use `blockExplorerUrl` also for non-EVM no? You did add it to `MULTICHAIN_PROVIDER_CONFIGS`

Reply:
Yes we can use it from the provider config